origarm_ros/Seg_ABL[9] segment
int16 segmentNumber
